libminifi/include/sitetosite/RawSocketProtocol.h

```diff
@@ -75,9 +75,9 @@ class RawSiteToSiteClient : public sitetosite::SiteToSiteClient {
     peer_ = std::move(peer);
     _batchSize = 0;
     _batchCount = 0;
-    _batchDuration = 0;
-    _batchSendNanos = 5000000000;  // 5 seconds
-    _timeOut = 30000;  // 30 seconds
+    _batchDuration = std::chrono::seconds(0);
+    _batchSendNanos = std::chrono::seconds(5);
+    _timeOut = std::chrono::seconds(30);  // 30 seconds
```

Comment:
The comment is redundant now, it can be removed.

Reply:
you are right, removed it in https://github.com/martinzink/nifi-minifi-cpp/commit/78f8658fde802c73a5965dd734a544acbc37dca3#diff-c1de4968ef4bf87b88a7ac2062a5abd667a9311affd42e8752a2531172e3357eR80